_projects/meta_review.md

```diff
@@ -0,0 +1,54 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch accepted to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and javascript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The goal of project meta-review is to build a meta-review track system similar to meta-moderation system that can track emoji responses to review comments. These responses would be collected, processed and displayed on [gh-board][https://github.com/coala/gh-board], which is a nice serverless kanban board.
+
+People including the author of pull request respond to comments by attaching emojis. Those emojis are called meta-review, or [reactions][https://developer.github.com/v4/enum/reactioncontent/]. THUMPS_UP and THUMPS_DOWN together with other reactions are used by people especially the author of pull request to publicly acknowledge or reject review comments. This provides feedback to reviewers and all other members.
+
+However, those reactions spread over all PRs and are not collected and analyzed. By tracking reactions, a nice feedback loop would be created for both senior and junior reviewers. Also, this information can be used as newcomer metrics.
+
+#### Milestones
+
+##### PREPARATION/BONDING
+
+- The applicant has proposed and merged a cEP of how meta-reviews could be tracked and handled.
```

Comment:
So I'd suggest that during this phase, 
1. we can make some mockups for the frontend you can get feedback
2. get acquainted with css designing a bit (materialize css) and data viz library like d3 (?)

Reply:
Also fix https://github.com/coala/gh-board/issues/13 ?
See https://github.com/coala/gh-board/issues/10#issuecomment-372038666

We're currently running the app on the browser on CI to fetch the issues from coala repositories and that isn't the most elegant thing https://github.com/coala/gh-board/blob/master/script/fetch-issues.js

I believe that'll help you in the long run.

Also see the React code a bit, you'll notice that it's still `React.createClass` which is deprecated, a small refactor to convert that into class component would be nice, and while we're at it might as well upgrade it to React v16.3 and further remove all the deprecations. But, I think all the refactoring is quite big, and not that really important/necessary for achieving this project. It is up to you.

Reply:
The refactoring will help you get familiar with the project and React in general (if you haven't already).

Reply:
I agree that getting the CI operational again, by creating our own issue json generator, should be a community bonding task.
And doing some JavaScript improvement tasks to remove deprecations will be good practise as working as a team.